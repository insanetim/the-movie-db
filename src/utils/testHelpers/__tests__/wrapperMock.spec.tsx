import { render } from '@testing-library/react'

import Wrapper from '../wrapperMock'

describe('mockWrapper component', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(
      <Wrapper>
        <div>test/children</div>
      </Wrapper>
    )

    expect(asFragment()).toMatchSnapshot()
  })
})
