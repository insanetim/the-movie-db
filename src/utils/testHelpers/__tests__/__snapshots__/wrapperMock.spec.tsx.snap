// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`mockWrapper component matches snapshot 1`] = `
<DocumentFragment>
  <div>
    test/children
  </div>
</DocumentFragment>
`;
